DataFormats/L1TMuon/src/EMTFHit.cc

```diff
@@ -1,4 +1,28 @@
 #include "DataFormats/L1TMuon/interface/EMTFHit.h"
 
 namespace l1t {
+
+  CSCDetId EMTFHit::CreateCSCDetId() const {
+    return CSCDetId( (endcap == 1) ? 1 : 2, station,    // For now, leave "layer" unfilled, defaults to 0.
+         (ring == 4) ? 1 : ring, chamber ); // Not sure if this is correct, or what "layer" does. - AWB 27.04.16
+  }
+
+  // // Not yet implemented - AWB 15.03.17
+  // RPCDetId EMTFHit::CreateRPCDetId() const {
+  //   return RPCDetId( endcap, ring, station, sector, rpc_layer, subsector, roll );
+  // }
+
+  CSCCorrelatedLCTDigi EMTFHit::CreateCSCCorrelatedLCTDigi() const {
+    return CSCCorrelatedLCTDigi( 1, valid, quality, wire, strip,
+         pattern, (bend == 1) ? 1 : 0,
+         bx + 6, 0, 0, sync_err, csc_ID );
+    // Unsure of how to fill "trknmb" or "bx0" - for now filling with 1 and 0. - AWB 27.04.16
```

Comment:
trknmb can be "1" or "2", depending on whether it is the first or second LCT. This is something I don't know either, but BX0 is always filled with 0 in the simulation.